servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractRestContents.java

```diff
@@ -129,27 +127,76 @@ public static Stream<ContentAndOperationType> contentAndOperationTypes() {
   public void verifyAllContentAndOperationTypes() throws BaseNessieClientServerException {
     Branch branch = createBranch("contentAndOperationAll");
 
+    List<ContentAndOperationType> contentAndOps =
+        contentAndOperationTypes().collect(Collectors.toList());
+
     CommitMultipleOperationsBuilder commit =
         getApi()
             .commitMultipleOperations()
             .branch(branch)
             .commitMeta(CommitMeta.fromMessage("verifyAllContentAndOperationTypes"));
-    contentAndOperationTypes()
-        .flatMap(
-            c ->
-                c.globalOperation == null
-                    ? Stream.of(c.operation)
-                    : Stream.of(c.operation, c.globalOperation))
-        .forEach(commit::operation);
-    commit.commit();
-
-    List<Entry> entries = getApi().getEntries().refName(branch.getName()).get().getEntries();
-    List<Entry> expect =
-        contentAndOperationTypes()
-            .filter(c -> c.operation instanceof Put)
-            .map(c -> Entry.builder().type(c.type).name(c.operation.getKey()).build())
-            .collect(Collectors.toList());
-    assertThat(entries).containsExactlyInAnyOrderElementsOf(expect);
+    contentAndOps.forEach(contentAndOp -> commit.operation(contentAndOp.operation));
+    Branch committed = commit.commit();
+
+    assertAll(
```

Comment:
do we really need to put this into an `assertAll`? The problem with this is that it's more difficult to debug imo when code is inside an `assertAll`, so I would prefer to not use `assertAll` here

Reply:
The opposite is actually true - it's easier to debug, because you see which operations actually fail.
Does it matter where you set the breakpoint?

Reply:
fair enough, however the argument about readability still holds. What's the big benefit using `assertAll` here?